src/utils/tankTableParser.ts

```
@@ -0,0 +1,325 @@
+  }
+}
+
+/**
+ * Main function to parse tank table from file
+ */
+export async function parseTankTableFile(file: File): Promise<TankTableParseResult> {
+  const extension = file.name.toLowerCase().split('.').pop();
+  
+  switch (extension) {
+    case 'csv':
+      const csvContent = await file.text();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)